content/docs/SUSHI/configuration/_index.md

```diff
@@ -86,6 +91,13 @@ In addition to the minimum configuration requirements shown above, most IG autho
       version: 3.1.0
   ```
 
+SUSHI also supports [extensions for converting between versions of FHIR](http://build.fhir.org/versions.html#extensions). To get extensions that represent elements from other versions of FHIR, a package of the form `hl7.fhir.extensions.<extension-version>:<package-version>` is used. The `<extension-version>` should be one of `r2`, `r3`, or `r4` to indicate which version of FHIR the element represented by the extension is defined in. The `<package-version>` represents which version of FHIR the extension will be used in. For an IG defined using FHIR R4, this would be `4.0.1`. As an example, if an author wanted to represent the `Patient.animal.species` [element](http://hl7.org/fhir/STU3/patient-definitions.html#Patient.animal.species) as defined in R3, the dependencies should be specified as:
+```yaml
+  dependencies:
+    hl7.fhir.extensions.r3: 4.0.1
+```
+And then reference the extension using the URL: `http://hl7.org/fhir/3.0/StructureDefinition/extension-Patient.animal.species`.
```

Comment:
This is very minor, but I think I'd make this a full sentence since it gets broken up by the code block. Maybe something as simple as

> An author can then reference the extension using the URL...